standalone-metastore/metastore-common/pom.xml

```diff
@@ -530,84 +499,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
```

Comment:
Same as above, we do not need integration test in `metastore-common`, maybe this plugin should be placed at `metastore-server`, which has aready been here.